src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -3429,14 +3446,17 @@ .maxstack  1
   IL_0018:  isinst     ""T""
   IL_001d:  unbox.any  ""T""
   IL_0022:  stloc.0
+  // sequence point: <hidden>
   IL_0023:  br.s       IL_0025
+  // sequence point: <hidden>
```

Comment:
> // sequence point: <hidden> [](start = 1, length = 28)

Do we need these hidden sequence points even if there are no when clauses?

Reply:
Can we also eliminate `br.s` instruction when there is no when clause?

---
In reply to: [185334656](https://github.com/dotnet/roslyn/pull/26138#discussion_r185334656) [](ancestors = 185334656)

Reply:
Yes, we need the hidden sequence point even without the when clause to avoid the appearance that we are executing the tail of the previous case block. It might be possible to eliminate it on the first case.

This branch is redundant because (1) There are no assignments to pattern variables in this case because the assignments have been optimized away, and (2) There is only a single case in the switch block.

---
In reply to: [185335043](https://github.com/dotnet/roslyn/pull/26138#discussion_r185335043) [](ancestors = 185335043,185334656)

Reply:
> Yes, we need the hidden sequence point even without the when clause to avoid the appearance that we are executing the tail of the previous case block

I guess the need arises for having the extra `br` there, correct?